### ISSUE TITLE
Show built-in extensions in extension search

### ISSUE THREAD

**Issue by digitarald** [microsoft/vscode#133767]
I had a few usertesting sessions were users wanted to set up VS Code for their languages and searched for CSS and JavaScript. The results don't serve the intent or communicate that those languages are built-in and fully featured.

To better explain which languages are shipped with VS Code, it could help if built-in extensions are displayed in search results. Related, _installed_ extensions are already shown in search results as well.

**Comment by isidorn** [comment]
@digitarald this makes sense and is good feedback, thanks!
in general I think the term "builtin extensions" should not be so visible to the users. Those are just extensions which are already installed. 

@sandy081 I guess the problem here is that most of the builtin extensions do not live on the Marketplace side, and MP is not even aware of them, right? And that's the reason why the search query does not have them?

**Comment by sandy081** [comment]
> I guess the problem here is that most of the builtin extensions do not live on the Marketplace side, and MP is not even aware of them, right? And that's the reason why the search query does not have them?

💯

**Comment by digitarald** [comment]
> @sandy081 I guess the problem here is that most of the builtin extensions do not live on the Marketplace side, and MP is not even aware of them, right? And that's the reason why the search query does not have them?

As an experiment, we could string-match (title, description) built-in extensions and prepend them to the search results? That should solve the heart of the problem (without a complex search algorithm matching).

**Comment by sandy081** [comment]
Yes we can do this hack but it looks search seems broken - filtering is inconsistent with MP and we have to some how filter those built in extensions exists in MP.
